core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1907,6 +1915,20 @@ class KafkaController(val config: KafkaConfig,
     }
 
     callback.apply(response)
+
+    // After we have returned the result of the `AlterIsr` request, we should check whether
+    // there are any reassignments which can be completed by a successful ISR expansion.
+    response.left.foreach { alterIsrResponses =>
+      alterIsrResponses.forKeyValue { (topicPartition, partitionResponse) =>
+        if (controllerContext.partitionsBeingReassigned.contains(topicPartition)) {
+          val isSuccessfulUpdate = partitionResponse.isRight
+          if (isSuccessfulUpdate) {
+            val assignment = controllerContext.partitionFullReplicaAssignment(topicPartition)
+            onPartitionReassignment(topicPartition, assignment)
```

Comment:
This old logic in processPartitionReassignmentIsrChange() does an info logging before calling onPartitionReassignment(). Should we do the same thing here?